README.mediawiki

```diff
@@ -124,7 +124,7 @@ Those proposing changes should consider that ultimately consent may rest with th
 | URI Scheme
 | Nils Schneider, Matt Corallo
 | Standard
-| Final
+| Replaced
```

Comment:
The color in the table depends on the Status (which we really should change, given that it doesn’t even render here), but anyway, the CI is failing because line 121 should now say "[…]#ffcfcf".